source/docs/controls-overviews/control-system-hardware.rst

```diff
@@ -74,17 +74,15 @@ The CTRE Voltage Regulator Module (VRM) is an independent module that is powered
   :alt: REV Radio Power Module
   :width: 500
 
-The [REV Radio Power Module](https://docs.revrobotics.com/rev-11-1856/) is designed to keep one of the most critical system components, the OpenMesh WiFi radio, powered in the toughest moments of the competition. The Radio Power Module eliminates the need for powering the radio through a traditional barrel power jack. Utilizing 18V Passive POE with two socketed RJ45 connectors, the Radio Power Module passes signal between the radio and roboRIO while providing power directly to the radio. After connecting the radio and roboRIO, easily add power to the Radio Power Module by wiring it to the low-current channels on the Power Distribution Hub utilizing the color coded push button WAGO terminals.
+The [REV Radio Power Module](https://docs.revrobotics.com/rev-11-1856/) was designed to keep one of the old OpenMesh WiFi radio, powered in the toughest moments of the competition. The Radio Power Module eliminates the need for powering the radio through a traditional barrel power jack. Utilizing 18V Passive POE with two socketed RJ45 connectors, the Radio Power Module passes signal between the radio and roboRIO while providing power directly to the radio. After connecting the radio and roboRIO, easily add power to the Radio Power Module by wiring it to the low-current channels on the Power Distribution Hub utilizing the color coded push button WAGO terminals.
```

Comment:
grammar.

I would tend to delete the RPM completely, as it isn't necessary with the VH-109, and can cause problems downstream with POE

Reply:
I was concerned about deleting it if people still use it to power their old radio but if deleting it is okay I am all for it.  I will do that.

Reply:
We still have the 2024 branch that would have the old radio